capybara-screenshot.gemspec

```diff
@@ -15,10 +15,12 @@ Gem::Specification.new do |s|
 
   s.rubyforge_project = "capybara-screenshot"
 
-  if RUBY_VERSION < "1.9"
+  if Gem::Version.new(RUBY_VERSION) < Gem::Version.new("1.9")
     s.add_dependency 'capybara', ['>= 1.0', '< 2']
-  else
+  elsif Gem::Version.new(RUBY_VERSION) <= Gem::Version.new("2.2.2")
```

Comment:
String instance of `"2.2.2"` is bigger than `"2.2.10"`, so I use Gem::Version to compare semvers